core/arch/aarch64/proc.c

```diff
@@ -45,6 +45,9 @@ proc_init_arch(void)
     num_simd_registers = MCXT_NUM_SIMD_SLOTS;
     num_opmask_registers = MCXT_NUM_OPMASK_SLOTS;
 
+    set_cache_line_size_using_ctr_el0(/* dcache_line_size= */ &cache_line_size,
```

Comment:
I would expect this to be named *get* not *set*: it's a query; it's not setting some persistent state.

Reply:
Done. I called it `set_` when it was writing the persistent `cache_line_size` var. But forgot to rename it later on.